src/ort_genai_c.cpp

```diff
@@ -214,6 +214,36 @@ void OGA_API_CALL OgaTokenizerDestroyStrings(const char** strings, size_t count)
   delete strings;
 }
 
+OgaResult* OGA_API_CALL OgaTokenizerEncodeBatchStrings(const OgaTokenizer* p, const OgaStrings* string_array, OgaSequences** out) {
+  OGA_TRY
+  auto& tokenizer = *reinterpret_cast<const Generators::Tokenizer*>(p);
+  auto& strings = *reinterpret_cast<const std::vector<std::string>*>(string_array);
+  *out = reinterpret_cast<OgaSequences*>(std::make_unique<Generators::TokenSequences>(std::move(tokenizer.EncodeBatch(strings))).release());
```

Comment:
What does EncodeBatch() returns and why move is necessary?

Reply:
Move was not necessary here. Removed it.